fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpResourceConverter.java

```diff
@@ -305,6 +289,32 @@ private Node getFrozenNodeByLabel(final String baseResourcePath, final String la
         }
     }
 
+    private Node getNode(final String baseResourcePath, final String label) throws RepositoryException {
```

Comment:
Just a future-aimed note that after Java 8,this is the kind of work that should be done by returning an [Optional](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html), not a value which might be null.